beacon-chain/sync/initial-sync/service.go

```diff
@@ -304,10 +252,52 @@ func (s *InitialSync) checkInMemoryBlocks() {
 	}
 }
 
+func (s *InitialSync) checkSyncStatus() bool {
```

Comment:
godoc comment? It isn't clear why this would return a bool or what it is checking from the signature